org-policy/.manifest

```diff
@@ -0,0 +1,3 @@
+{
+    "roots": ["diamondlightsource"]
```

Comment:
Are we going with `diamond/policy`?

Reply:
I wanted to keep this the minimal change for getting 2 bundles loading, so not changing the packages and roots of the policy bundle. That occurs in the non-dummy policy change